ios/AppDelegate.swift

```diff
@@ -40,7 +40,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     // firebase used for app distribution.
     // FirebaseApp.configure(options: options)
     
-    registerForPushNotifications()
+//    registerForPushNotifications()
```

Comment:
May be we should remove this completely? 🤔

Reply:
Yes, removed now.